modules/oadp-features.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/oadp-features-plugins.adoc
+
+[id="oadp-features_{context}"]
+= OADP features
+
+OpenShift API for Data Protection (OADP) supports the following features:
+
+Backup::
+You can back up all resources in your cluster or you can filter the resources by type, namespace, or label.
++
+OADP backs up Kubernetes objects and internal images by saving them as an archive file on object storage. OADP backs up persistent volumes (PVs) by creating snapshots with the native snapshot API or with the Container Snapshot Interface (CSI).
++
+For storage providers that do not support snapshots, OADP backs up resources and PV data with Restic.
+
+Restore::
+You can restore resources and PVs from a backup. You can restore all objects in a backup or filter the restored objects by namespace, PV, or label.
+
+Schedule::
+You can schedule backups at specified intervals.
+
+Hooks::
+You can use hooks to run commands in a container on a pod, for example, `fsfreeze` to freeze a file system. You can configure a hook to run before or after a backup or restore. Restore hooks can run in an init container or in the application container.
```

Comment:
Just a suggestion: 
"You can use hooks to run commands in a container on a pod, for example, `fsfreeze` to freeze a file system."
Something like:
You can use hooks to run commands automatically when container or pod registers an event. For example, you can set up a hook to freeze a file system when such-and-so happens.

Reply:
You can't configure a hook to run when an event is registered. I'm not sure how you got this interpretation.